dashboard/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from OneApply.constants import UserType
+
+
+def student(request):
+    context = {
+        "constant_ut_student": UserType.STUDENT,
+        "constant_ut_adminStaff": UserType.ADMIN_STAFF,
```

Comment:
Is there a particular reason why the admin staff user type is being mentioned in the student view?

Reply:
either combine this into one view or make the changes as Patryk has suggested

Reply:
> Is there a particular reason why the admin staff user type is being mentioned in the student view?

i think remain it for now. Cause maybe later we might need these constants. Like if you integrating your page into dashboard, the page may require those constants.

Reply:
I am not questioning the use of the constants. I am just wondering why you chose to include the admin_staff user type for the student dashboard.

Reply:
> I am not questioning the use of the constants. I am just wondering why you chose to include the admin_staff user type for the student dashboard.

If there is a  tab page, which has buttons one for student and one for admission staff, then it will required both two constants. And if I didn't include those constants in advance, then there will be an error. Since I dont know the other tab pages' code, I think include both are safe way for our integrating and developing